Material.Blazor.Website/Shared/MainLayout.razor

```diff
@@ -67,7 +67,7 @@
                  NavIcon="menu"
                  ScrollTarget="#main-content"
                  TopAppBarType="@MBTopAppBarType.Dense"
-                 class="mdc-top-app-bar--short-has-action-item mdc-elevation--z5">
+                 class="app-bar mdc-top-app-bar--short-has-action-item mdc-elevation--z5">
```

Comment:
This was a bug: `app-bar` was added in the Material.Blazor project, but should be here in the website only. The CSS class is defined for the website only.